packages/react/package.json

```
@@ -110,7 +110,7 @@
     "lint-staged": "^7.0.0",
     "prettier": "^1.4.4",
     "prompt": "^1.0.0",
-    "react-test-renderer": "^16.13.1",
+    "react-test-renderer": "^17.0.1",
```

Hi @antonhalim, sorry for the late reply! Just one thing, what do you think of relaxing the version a bit, like, allowing React v16 _or_ v17? Something like:
```suggestion
    "react-test-renderer": "^16.13 || ^17.0.1",
```
My point is that we don't want to force folks to upgrade to React 17 just to get features/fixes they need after this change is made.

---

Oh yeah, for sure, thanks!